docs/install.md

```diff
@@ -200,68 +200,89 @@ Another option to run containerlab on ARM-based Macs is to use Docker in Docker
     === "Dockerfile"
 
         ```Dockerfile
-        # The devcontainer will be based on Python 3.9
+        # The devcontainer will be based on debian bullseye
         # The base container already has entrypoint, vscode user account, etc. out of the box
-        FROM mcr.microsoft.com/devcontainers/python:0-3.9-bullseye
+        FROM mcr.microsoft.com/vscode/devcontainers/base:bullseye
 
         # containelab version will be set in devcontainer.json
         ARG _CLAB_VERSION
 
+        # Set permissions for mounts in devcontainer.json
+        RUN mkdir /home/vscode/.vscode-server/
+        RUN mkdir /home/vscode/.vscode-server/bin
+        RUN chown -R vscode:vscode /home/vscode/.vscode-server
+
         # install some basic tools inside the container
         # adjust this list based on your demands
         RUN apt-get update \
-            && apt-get install -y --no-install-recommends \
+            && apt-get upgrade -y
+            apt-get install -y --no-install-recommends \
```

Comment:
@neilschark 
I wouldn't do the upgrade here, it will take quite some time and will leave you with a different base OS more frequently then package updates.
Not sure it is worth it, but I may be wrong.

What seems to be clearly missing here though is `&&` after upgrade

Reply:
I don't think time is a problem here, as the creation of the dev container from scratch depending on the added feature already takes several minutes if you have a slow Internet connection. But after the first building everything is cached, so it doesn't matter anymore, as it is not run again until you pull a clean base image.

I get you second point though, it makes the image possibly different. I don't think it matters much, as for example the docker in docker feature is added with the 'latest' tag, so you will have this problem anyway.

In my experience the base image of debian is updated less enough, that you get some updates in this step regularly. I would guess that the image provided by Microsoft which is the base here is updated even less, so updates with security fixes would likely occur.

But at the and it is not a mayor difference either way, so I would leave it up to the project (and therefore you) to decide.

Just to have it complete I just add the missing '&&' you mentioned. Simply either accept the PR like it is now or tell me to remove the update line, I am fine with it both ways.